components/grid/export/csv.md

```diff
@@ -138,42 +140,66 @@ You can programmatically invoke the export feature of the Grid, by using the fol
     </GridExport>
 
     <GridColumns>
-        <GridColumn Field="@nameof(SampleData.ProductId)" Title="ID" />
-        <GridColumn Field="@nameof(SampleData.ProductName)" Title="Product Name" />
-        <GridColumn Field="@nameof(SampleData.UnitsInStock)" Title="In stock" />
-        <GridColumn Field="@nameof(SampleData.Price)" Title="Unit Price" />
-        <GridColumn Field="@nameof(SampleData.Discontinued)" Title="Discontinued" />
-        <GridColumn Field="@nameof(SampleData.FirstReleaseDate)" Title="Release Date" />
+        <GridColumn Field="@nameof(SampleData.ProductId)" Title="ID" Width="100px" />
+        <GridColumn Field="@nameof(SampleData.ProductName)" Title="Product Name" Width="300px" />
+        <GridColumn Field="@nameof(SampleData.UnitsInStock)" Title="In stock" Width="100px" />
+        <GridColumn Field="@nameof(SampleData.Price)" Title="Unit Price" Width="200px" />
+        <GridColumn Field="@nameof(SampleData.Discontinued)" Title="Discontinued" Width="100px" />
+        <GridColumn Field="@nameof(SampleData.FirstReleaseDate)" Title="Release Date" Width="300px" />
```

Comment:
Setting pixel widths to all columns should be done only if you want to have a horizontal scrollbar. Is this the case? If the columns overflow the docs REPL, then consider removing some of them instead.